### DISCUSSION TITLE
اضافه کردن قابلیت لول بندی

### DISCUSSION THREAD

**Discussion by MoSi8160** [kianahmadian/python-bale-bot#60] [Ideas] [answered]
سلام خسته نباشید 
یه قابلیتی که به شدت نیازه اینه که بتونیم سطح کاربر رو مشخص کنیم

مثلاً هنگام شروع ربات فرد در سطح ۱ هست و ۴ دکمه جلوش هست
بعد اگر روی دکمه ۳ بزنه سطح عوض بشه و چهارتا دکمه جدید جلوش باشه 
نمیدونم تونستید متوجه منظورم بشید یا خیر

**Comment by kian-ahmadian** [comment, marked as answer]
سلام عزیز وقت بخیر
از پیشنهاد تون متشکرم، در آپدیت های بعدی قابلیتی اضافه میشه تا بتونید اطلاعاتی و در یوزر ها و چت ها کش کنید تا بتونین در حین کار از اون ها استفاده کنید
